lib/services/jellyfin_api_helper.dart

```diff
@@ -179,104 +272,79 @@ class JellyfinApiHelper {
   Future<PublicUsersResponse> loadPublicUsers() async {
-    Response response;
-
     // Some users won't have a password.
     if (baseUrlTemp == null) {
       return PublicUsersResponse(users: []);
     }
 
-    response = await jellyfinApi.getPublicUsers();
+    var response = await jellyfinApi.getPublicUsers();
 
-    if (response.isSuccessful) {
-      PublicUsersResponse publicUsersResult = PublicUsersResponse(
-        users: (response.body as List<dynamic>).map((userJson) => UserDto.fromJson(userJson)).toList(),
-      );
+    PublicUsersResponse publicUsersResult = PublicUsersResponse(
+      users: (response as List<dynamic>)
+          .map((userJson) => UserDto.fromJson(userJson))
+          .toList(),
+    );
 
-      return publicUsersResult;
-    } else {
-      return Future.error(response);
-    }
-    
+    return publicUsersResult;
   }
 
   /// Check if server has Quick Connect enabled.
   Future<bool> checkQuickConnect() async {
-    Response response;
-
-    response = await jellyfinApi.getQuickConnectState();
-
-    return response.isSuccessful && response.body;
+    var response = await jellyfinApi.getQuickConnectState();
+    return response;
   }
 
   /// Initiate a Quick Connect request.
   Future<QuickConnectState> initiateQuickConnect() async {
-    Response response;
-
-    response = await jellyfinApi.initiateQuickConnect();
+    var response = await jellyfinApi.initiateQuickConnect();
 
-    if (response.isSuccessful) {
-      QuickConnectState quickConnectState =
-          QuickConnectState.fromJson(response.body);
-
-      return quickConnectState;
-    } else {
-      return Future.error(response);
-    }
+    QuickConnectState quickConnectState = QuickConnectState.fromJson(response);
 
+    return quickConnectState;
   }
 
   /// Update the Quick Connect state.
-  Future<QuickConnectState?> updateQuickConnect(QuickConnectState quickConnectState) async {
-    Response response;
-
-    response = await jellyfinApi.updateQuickConnect(secret: quickConnectState.secret ?? "");
+  Future<QuickConnectState?> updateQuickConnect(
+      QuickConnectState quickConnectState) async {
+    var response = await jellyfinApi.updateQuickConnect(
+        secret: quickConnectState.secret ?? "");
 
-    if (response.isSuccessful) {
-      QuickConnectState quickConnectState =
-          QuickConnectState.fromJson(response.body);
+    QuickConnectState newQuickConnectState =
+        QuickConnectState.fromJson(response);
 
-      return quickConnectState;
-    } else {
-      return Future.error(response);
-    }
+    return newQuickConnectState;
   }
 
   /// Authenticates a user using Quick Connect and saves the login details
-  Future<void> authenticateWithQuickConnect(QuickConnectState quickConnectState) async {
-    Response response;
-
-    response = await jellyfinApi.authenticateWithQuickConnect({"Secret": quickConnectState.secret ?? ""});
-
-    if (response.isSuccessful) {
```

Comment:
Did you remove that check because `jellyfinApi` methods throw an error for >=400 status codes?

Reply:
I bumped the version of chopper, and the new version defaulted to returning the body and throwing an error on unsuccessful response codes instead of returning a response like the old version, so I just went with that because we were only re-throwing the response as an error anyway.

Reply:
I see. Was the Chopper upgrade a major version bump?

Reply:
I believe it was 7.1.0 that broke everything, so in theory no.  I'm thinking the way we use dynamic for all the api return types threw things off, because the generator saw we would accept the bodies even though everything was actually expecting a response.

Reply:
Yeah I've had problems with 7.1.1 too, if that is the fix for the `type '_Map<String, dynamic>' is not a subtype of type 'Response<dynamic>'`, that would be nice!